db/migrate/20230227123942_create_workspaces.rb

```diff
@@ -0,0 +1,14 @@
+class CreateWorkspaces < ActiveRecord::Migration[7.0]
+  def change
+    create_table :workspaces do |t|
+      t.string :name
+      t.float :price
+      t.string :address
+      t.string :photo
```

Comment:
Ultimately we will delete this field and use ActiveStorage, but for now this is fine